source/_components/yessssms.markdown

```diff
@@ -31,6 +47,7 @@ notify:
     username: YOUR_PHONE_NUMBER
     password: YOUR_PASSWORD
     recipient: PHONE_NUMBER_TO_NOTIFY
+    provider: MVNO_PROVIDER
```

Comment:
if `provider` is not required then please remove.

Reply:
well, it's an optional parameter, but you need it if you want to use the option. I removed it.